dsgrid/tests/make_us_data_registry.py

```diff
@@ -56,9 +56,6 @@ def make_test_data_registry(
     dataset_path = Path(dataset_path)
     path = create_local_test_registry(registry_path)
     dataset_dir = Path("datasets/sector_models/comstock")
-    project_dimension_mapping_config = src_dir / "dimension_mappings.toml"
-    dimension_mapping_config = src_dir / dataset_dir / "dimension_mappings.toml"
-    dimension_mapping_refs = src_dir / dataset_dir / "dimension_mapping_references.toml"
```

Comment:
While it still exists we should continue to test the old, slow way of registering everything. Is that still happening?

Reply:
There are tests that explicitly call dimension_manager.register_from_file and dimension_mapping_manager.register_from_file, but we do not test that entire workflow. The code paths are shared and so I didn't think it was necessary. But I'm open to it if you think we need a full example.

Reply:
I think that would be best/least regrests to have two parallel examples, one with each workflow.